packages/search-adapter/src/empathy/__tests__/response-mappers/empathy-facet-mapper.spec.ts

```diff
@@ -47,21 +47,31 @@ afterEach(() => {
 describe('Empathy Hierarchical Facet', () => {
   it('maps a hierarchical facet', () => {
     const mappedFacet = mapFacet(hierarchicalRawFacet, {} as HierarchicalFacet, emptyContext) as HierarchicalFacet;
-
```

Comment:
There is an error in the line 39

![image](https://user-images.githubusercontent.com/3502075/128664263-8c6f9000-5f5d-40d9-bcc0-ada05b4e8895.png)

Regarding `tslib`, please take a look.